<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginDialog</class>
 <widget class="QDialog" name="loginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="usrlineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdlineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="usrlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="pwdlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="hostlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>77</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Host</string>
   </property>
  </widget>
  <widget class="QLabel" name="portlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>77</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addrlineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="portlineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>loginDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
